docs/docs-source/docs/modules/get-started/pages/install-cloudflow.adoc

```diff
@@ -0,0 +1,322 @@
+
+If you plan to write applications that utilize Spark you will need to install the Spark operator before deploying your Cloudflow application.
+
+The Spark operator used is currently maintained by Lightbend and is presently the only Cloudflow compatible version.
+
+Add the Spark Helm chart repository.
+
+  helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+
+As we have seen before, we need to update the Helm repo before installing Spark.
+  
+  helm repo update
+
+Before installing the Helm chart, we will need to prepare a `value.yaml` file, it overrides the default values in the Spark Helm chart.
+
+----
+cat > spark-values.yaml << EOF
+rbac:
+  create: true
+serviceAccounts:
+  sparkoperator:
+    create: true
+    name: cloudflow-spark-operator
+  spark:
+    create: true
+    name: cloudflow-spark
+
+enableWebhook: true
+enableMetrics: true
+
+controllerThreads: 10
+installCrds: true
+metricsPort: 10254
+metricsEndpoint: "/metrics"
+metricsPrefix: ""
+resyncInterval: 30
+webhookPort: 8080
+sparkJobNamespace: ""
+operatorImageName: "lightbend/sparkoperator"
+operatorVersion: "2.0.7-cloudflow-spark-2.4.5-1.1.2-scala-2.12"
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: cloudflow-webhook
+  labels:
+    app.kubernetes.io/name: sparkoperator
+spec:
+  selector:
+    app.kubernetes.io/name: sparkoperator
+    app.kubernetes.io/version: 2.0.7-cloudflow-spark-2.4.5-1.1.2-scala-2.12
+EOF
+----
+
+Now we can install the Spark operator using Helm. Note that we use a specific version fo the Spark operator Helm chart.
+
+  helm install spark-operator incubator/sparkoperator --namespace cloudflow --values="spark-values.yaml"  --version "0.6.7"
+
+IMPORTANT: If you are installing the Spark operator on *OpenShift*, you need to apply the following `rolebinding` to the cluster.
+
+----
+cat > spark-rolebinding.yaml << EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: system:openshift:scc:anyuid
+  namespace: cloudflow
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: system:openshift:scc:anyuid
+subjects:
+- kind: ServiceAccount
+  name: cloudflow-spark
+  namespace: cloudflow
+EOF
+----
+
+To apply the `spark-rolebinding.yaml` file execute the following command. Note that this has *only* to be performed if you are installing Cloudflow on Openshift.
+
+  oc apply -f spark-rolebinding.yaml
+
+After the Spark operator installation is completed, we need to adjust its webhook configuration.
+
+----
+$ kubectl get pods -n cloudflow
+NAME                                                READY   STATUS    RESTARTS   AGE
+cloudflow-operator-6b7d7cbdfc-xb6w5                 1/1     Running   0          109s
+spark-operator-sparkoperator-56d6ffc8cd-cln67       1/1     Running   0          109s
+----
+
+For this, we will create yet another YAML file that we will apply to the cluster with `kubectl`.
+----
+cat > spark-mutating-webhook.yaml << ‘EOF2’
```

Comment:
Looks like these quotes are a problem, rather use normal single quotes

Reply:
ok, I tested this also on Bash, weird.

Reply:
Now checked, these are normal single quotes in the source, they get rendered as something else here, not sure why. We have to run these instructions on the final rendered output also it seems.